compiler/ast/ast_parsed_types.nim

```diff
@@ -175,6 +174,7 @@ type
     pnkExportStmt
     pnkExportExceptStmt
     pnkConstSection
+    pnkTypeSection
```

Comment:
```suggestion
    pnkTypeSection
    pnkConstSection
```

const, let, and var sections are very similar, unlike type sections, type section shouldn't break them up.